_gsocblogs/2022/blog_ROOT_Neelshah.md

```diff
@@ -0,0 +1,366 @@
+---
+project: ROOT
+title: Inference of Deep Learning models in TMVA/SOFIE
+author: Neel Shah
+photo: blog_authors/NeelShah.jpg 
+date: 24.07.2022 
+year: 2022 
+layout: blog_post 
+logo: ROOT-logo.png 
+intro: |
+Mid Evaluation Report of GSOC 2022 for the project :- Inference of Deep Learning Models in TMVA/SOFIE.
```

Comment:
```suggestion
Mid Evaluation Report of GSOC 2022 for the project\: Inference of Deep Learning Models in TMVA/SOFIE.
```

Reply:
The sequence `:-` does not escape the `:` and does not render. Here you should put a short intro, not a title.

Reply:
Done.